Sources/Interaction/Widgets/PiecewiseGaussianWidget/index.js

```diff
@@ -1050,7 +1057,7 @@ export function extend(publicAPI, model, initialValues = {}) {
     'colorTransferFunction',
     'backgroundImage',
   ]);
-  macro.get(publicAPI, model, ['size', 'canvas']);
+  macro.get(publicAPI, model, ['size', 'canvas', 'selectedGaussian']);
```

Comment:
You will override your `getSelectedGaussian` method if you do that so the resulting `getSelectedGaussian()` will return the index.

Reply:
Good call. Removing...